From f5d371d6ecdbb84acf0de34cd80ddaee2493ca72 Mon Sep 17 00:00:00 2001
From: Daniel Kolesa <d.kolesa@samsung.com>
Date: Thu, 11 Jun 2020 02:32:07 +0200
Subject: [PATCH 6/7] ecore: use standard LC_ALL instead of __LC_ALL in systemd
 module

The former is a POSIX name, the latter is non-standard. I don't
know why the latter was used, considering glibc literally has
just #define LC_ALL __LC_ALL, but change it and unbreak build
on musl and other systems.
---
 src/modules/ecore/system/systemd/ecore_system_systemd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git src/modules/ecore/system/systemd/ecore_system_systemd.c src/modules/ecore/system/systemd/ecore_system_systemd.c
index 78e3185287..ab19ff36cd 100644
--- src/modules/ecore/system/systemd/ecore_system_systemd.c
+++ src/modules/ecore/system/systemd/ecore_system_systemd.c
@@ -163,7 +163,7 @@ static void _locale_get(void *data EINA_UNUSED, const Eldbus_Message *msg,
 
         setenv(type, value, 1);
      }
-   setlocale(__LC_ALL, "");
+   setlocale(LC_ALL, "");
 
  end:
    ecore_event_add(ECORE_EVENT_LOCALE_CHANGED, NULL, NULL, NULL);
-- 
2.27.0

